src/main/java/com/xiaomi/infra/pegasus/rpc/async/TableHandler.java

```diff
@@ -211,8 +226,25 @@ void onRpcReply(
       int tryId,
       ReplicaConfiguration cachedHandle,
       long cachedConfigVersion) {
-    client_operator operator = round.getOperator();
+    // judge if it is the first response
+    if (round.isSuccess) {
+      return;
```

Comment:
`return` will result in the all follow  logics is not executed.(for example, the log will not be printed)

Reply:
yes, we only need the response which is return first. others will be ignored, or the logics will be a mess

Reply:
I mean if success, you still need excute the follow logic. for example, the logic:
```
switch (operator.rpc_error.errno) {
      case ERR_OK:
        round.thisRoundCompletion();
        return;
.....
```
shouldn't be ignored.

Reply:
> the response which is return

`round.thisRoundCompletion()` can result callback and return the response. "only return" will not get any response.